client/src/components/History/CurrentHistory/HistoryOperations/SelectionOperations.vue

```diff
@@ -42,6 +43,19 @@
                 @click="buildCollectionFromRules">
                 <span v-localize>Build Collection from Rules</span>
             </b-dropdown-item>
+            <b-dropdown-divider />
+            <b-dropdown-item v-b-modal:change-dbkey-of-selected-content data-description="change reference gnome">
+                <span v-localize>Change reference Gnome</span>
```

Comment:
```suggestion
                <span v-localize>Change reference Genome</span>
```
🤣

Reply:
I'm also gonna say this shouldn't be genome centered. the `dbkey` can refer to anything.

Reply:
:laughing: :+1:

Reply:
oh sorry, the previous comment didn't show up until I committed the suggestion.
Should we use `Change dbkey` or `Change database key`?

Reply:
<img width="325" alt="Screenshot 2022-05-26 at 13 26 17" src="https://user-images.githubusercontent.com/6804901/170479090-d29fb290-5176-4d86-a2b9-cc75dfb61da6.png">
is how the single-item menu looks like ...

Can we do `Database/Build` for now and defer any decision to rename this for later ?